.github/workflows/pr-welcome.yml

```diff
@@ -25,3 +38,14 @@ jobs:
               issue_number: ${{ github.event.number }},
               body: 'Thank you for your contribution! 🙏 We will review your PR as soon as possible.\n\n\nWhile you are waiting, make sure to:\n\n\n- Add an entry in [our changelog](https://github.com/kedacore/keda/blob/main/CHANGELOG.md) in alphabetical order and link related issue\n- Update the [documentation](https://github.com/kedacore/keda-docs), if needed\n- Add unit & [e2e](https://github.com/kedacore/keda/blob/main/tests/README.md) tests for your changes\n- GitHub checks are passing\n- Is the DCO check failing? Here is [how you can fix DCO issues](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what)\n\n\nLearn more about:\n- Our [contribution guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md)'
             });
+      - name: 'Apply review required label'
+        uses: actions/github-script@v6
+        if: "!github.event.pull_request.draft"
+        with:
+          script: |
+            github.rest.issues.addLabels({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              labels: ["requires-pr-review"]
```

Comment:
Is this really needed?

Reply:
And I think that the label will stay there, even if I do a review.

Reply:
Yes, that is coming in a other PR coming later on. see attached issue where it's also for PR reviewers to get our intention instead of just posting comments which you cannot find when looking through PRs unless it's unread which is bad.

Reply:
Let's proceed with this, but I would like  the label to be automatically removed after a review from maintainer. And not something we need to do manually.

Reply:
Done, but will need to merge before testing